scripts/build_server_local

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I think maintaining both a local and a docker build is going to be difficult. We should probably decide on one or the other. Let's have a chat about pros and cons when @tiziano88 is back

Reply:
I'll drop this commit from the PR and we can discuss later.  (I've also realized it needs more tweaking anyhow &ndash; switching between client and server builds invalidates the Bazel cached outputs, as they use different toolchains.)

However, I will note that the speed difference between a local build and a Docker build is quite large (60x)&hellip;
```console
# Second run with no changes in between
% time ./scripts/build_server_docker  > /dev/null 2>&1
real	0m12.643s
user	0m0.103s
sys	0m0.096s
% time ./scripts/build_server_local > /dev/null 2>&1
real	0m0.213s
user	0m0.052s
sys	0m0.026s
```